distribution/test/itests/pom.xml

```diff
@@ -31,9 +31,12 @@
         <module>test-itests-dependencies-app</module>
     </modules>
     <properties>
-        <pax.exam.version>4.9.2</pax.exam.version>
+        <!-- Copied from ops4j/org.ops4j.pax.exam2/pom/pom.xml
```

Comment:
We should include a version number in this comment, like we did [here](https://github.com/codice/ddf/blob/master/platform/platform-app/src/main/resources/features.xml#L36).